lib/consult/template.rb

```diff
@@ -34,7 +34,7 @@ def dest
     end
 
     def should_render?
-      (@config[:environments] == 'all' || @config[:environments].include?(Consult.env)) && expired?
+      (@config[:environments] == 'all' || [@config[:environments]].flatten.include?(Consult.env)) && expired?
```

Comment:
This one is interesting. `@config[:environments]` should already be an array if we format our YAML properly, which I don't think we currently are. In other words:

```yml
environments: ['production', 'staging']
```

or

```yml
environments:
  - production
  - staging
```

Should both work. We should modify consult to assume standard YAML conventions here.

Reply:
I don't have a problem with supporting both `environments: <string>` and `environments: <array>`. It's more ergonomic for single environments and this is a safe, low energy change.

However, I still think that `consult.yml` _should_ be refactored with environment specific blocks, rails style. I think we can figure out how to do the right thing with that in non rails situations, and then this particular question becomes moot.

Reply:
This is moot with https://github.com/veracross/consult/pull/11